### DISCUSSION TITLE
Is it possible to add classes to specific notes or beats?

### DISCUSSION THREAD

**Discussion by carlows** [CoderLine/alphaTab#784] [Q&A] [answered]
Hi there, thanks for the amazing tool :)

I'm wondering if it's possible to target specific notes or beats and add custom classes to them. Or in general, being able to get the note or beat element and attach a reference that I can target through CSS.

There's already a class that gets appended to the notes whenever the cursor is over the notes, but I would like to be able to target specific notes that I get from the data model.

Is there a way to do this?

**Comment by Danielku15** [comment, marked as answer]
This topic has already been discussed 
https://github.com/CoderLine/alphaTab/discussions/613 and https://github.com/CoderLine/alphaTab/issues/291 and https://github.com/CoderLine/alphaTab/issues/213 and https://github.com/CoderLine/alphaTab/issues/248

**Reply by carlows** [reply]
Thanks, I think `boundsLookup` is enough for what I want to do 😄

**Comment by Danielku15** [comment]
@carlows 
Better late then never 😉 
Coloring is now available in the 1.5.0 pre-release versions. Any feedback is appreciated. 

https://next.alphatab.net/docs/guides/coloring
